isotope/service/main.go

```diff
@@ -0,0 +1,98 @@
+// Copyright 2018 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this currentFile except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"net/http"
+	"os"
+	"path"
+	"runtime"
+
+	"istio.io/tools/isotope/convert/pkg/consts"
+	"istio.io/tools/isotope/service/pkg/srv"
+	"istio.io/tools/isotope/service/pkg/srv/prometheus"
+	"istio.io/fortio/log"
+)
+
+const (
+	promEndpoint    = "/metrics"
+	defaultEndpoint = "/"
+)
+
+var (
+	serviceGraphYAMLFilePath = path.Join(
+		consts.ConfigPath, consts.ServiceGraphYAMLFileName)
+
+	// Set by the Python script when running this through Docker.
+	maxIdleConnectionsPerHostFlag = flag.Int(
+		"max-idle-connections-per-host", 0,
+		"maximum number of TCP connections to keep open per host")
+)
+
+func main() {
+	flag.Parse()
+
+	log.SetLogLevel(log.Info)
```

Comment:
that's the default already, why do you force it / override the possible flag ?

you could consider adding
```
	if *bincommon.QuietFlag {
		log.SetLogLevelQuiet(log.Error)
	}
```
like in fortio_main and fortio curl

Reply:
I think it's good to be explicit. I'd rather not add more other packages if I don't need them.

Reply:
Actually, allowing a flag to set the level is a good idea. Removed the forceful setting.

Reply:
thanks for having fixed it